stream-chat-android/src/main/java/com/getstream/sdk/chat/ChatUI.kt

```diff
@@ -84,6 +85,10 @@ public class ChatUI internal constructor(
             this.strings = strings
         }
 
+        public fun withImageRequestHeaders(imageRequestHeaders: Map<String, String>): Builder = apply {
+            StreamImageLoader.instance().imageRequestHeaders = imageRequestHeaders
+        }
+
```

Comment:
We probably don't need to add this to the old implementation

Reply:
We do, it will be used in the old one